timesketch/api/v1/resources/sigma.py

```diff
@@ -90,3 +93,39 @@ def get(self, rule_uuid):
                 HTTP_STATUS_CODE_NOT_FOUND, 'No sigma rule found with this ID.')
 
         return return_rule
+
+class SigmaByTextResource(resources.ResourceMixin, Resource):
+    """Resource to get list of Sigma rules."""
+
+    @login_required
+    def post(self):
+        """Handles POST request to the resource.
+
+        Returns:
+            JSON sigma rule
+        """
+        form = forms.SigmaRuleForm.build(request)
+        # optional: do we want to have a title at this point?
+        title = form.title.data
+        content = form.content.data
```

Comment:
I'm missing some input validation here

Reply:
I have rewritten that now, so not sure what kind of validation you have in mind